src/showcases/components/styled/styledComponentVariants.component.tsx

```diff
@@ -1,23 +1,27 @@
 import React from 'react';
 import { TouchableOpacity } from 'react-native';
-import { Interaction, styled } from '@ui-kitten/components';
+import { EvaProp, Interaction, styled } from '@ui-kitten/components';
+
+interface IProps {
```

Comment:
Please rename non-exportable interface to `Props`. According to our styleguide we avoid `I` prefixes